force dnsmasq to listen only on eth1 in the minimal plan

yuvalk opened the pull request.

yuvalk committed:
force dnsmasq to listen only on eth1 in the minimal plan

karmab commented:
this means the node only answers for dns on a dedicated nic, and we dont want that

yuvalk commented:
Why not? dnsmasq here should really work only on the private lan, not the
other one, which is needed only to install packages on that node, etc

On Sat, May 8, 2021, 12:38 Karim Boumedhel ***@***.***> wrote:

> this means the node only answers for dns on a dedicated nic, and we dont
> want that
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/karmab/kcli-upi-helper/pull/5#issuecomment-835247334>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AASLA27GNWXHI6I6T66C3IDTMUA7TANCNFSM44JIXNCA>
> .
>

karmab commented:
fair enough but as it is, it breaks the plan because being able to resolve dns from eth0 is what allows to install packages such as radvd after setting dnsmasq

karmab commented:
i guess the issue is this line, used to force the node to use its own dns
echo nameserver $(hostname -I | cut -d' ' -f1) > /etc/resolv.conf
if we force to eth1, then i guess we should use eth1 address here

yuvalk commented:
Well, why do we change minimal-installer resolve.conf at all?

On Sat, May 8, 2021, 13:00 Karim Boumedhel ***@***.***> wrote:

> i guess the issue is this line, used to force the node to use its own dns
> echo nameserver $(hostname -I | cut -d' ' -f1) > /etc/resolv.conf
> if we force to eth1, then i guess we should use eth1 address here
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/karmab/kcli-upi-helper/pull/5#issuecomment-835259647>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AASLA26L4PUYKPI5K4YSN53TMUDTBANCNFSM44JIXNCA>
> .
>

yuvalk committed:
add option to disable default route in minimal plan

this is needed in cases where there's a secondary nic with default
route,
mainly on installer VMs that provide gateway to the 'real' network
